ui/src/router/components/form/components/router_config/rule_card/RuleCard.js

```diff
@@ -81,6 +82,21 @@ export const RuleCard = ({
         </EuiFlexItem>
       </EuiFlexGroup>
 
+      <EuiFormRow
+        label="Name *"
+        isInvalid={!!get(errors, "name")}
+        error={get(errors, "name")}
+        fullWidth>
+        <EuiFieldText
```

Comment:
This got me thinking - when we get to the UI changes for the Traffic rules, we could add the rule name to the existing routes view as well.